extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -193,7 +193,7 @@ describes.realWin('amp-story', {
         });
   });
 
-  // TODO(newmuis/amphtml-story#187): Re-enable this test.
+  // TODO(#187): Re-enable this test.
```

Comment:
We want this to be #11639, here and below

Reply:
Oops, corrected.